pages/recipe/[id]/edit.js

```diff
@@ -21,9 +23,12 @@ export default function EditRecipe({ user }) {
     });
 
     if (response.ok) {
-      router.back();
+      mutate();
```

Comment:
I don't think you need mutate here, since you use router.push to navigate back to your homepage (which should trigger a revalidation, since there is also a fetch with useSWR there)